CovertInfiltration/Src/CovertInfiltration/Classes/X2EventListener_Infiltration_UI.uc

```diff
@@ -510,19 +513,50 @@ static function EventListenerReturn UpdateResources(Object EventData, Object Eve
 	{
 		if (
 			CurrentScreen == CovertActions ||
-			(CovertActions.SSManager != none && CovertActions.SSManager.ShouldShowResourceBar() && CurrentScreen.IsA(class'UISquadSelect'.Name))
-		) {
+			(CovertActions.SSManager != none && CovertActions.SSManager.ShouldShowResourceBar() && CurrentScreen.IsA(class'UISquadSelect'.Name)))
```

Comment:
This also shows on Squad Select for vanilla CAs, but we don't want that (since you can already see that by opening the personnel list)